src/modals/JoinPool/index.tsx

```diff
@@ -64,7 +64,7 @@ export const JoinPool = () => {
 
   // tx to submit
   const getTx = () => {
-    const tx = null;
+    let tx = null;
```

Comment:
Why?

Reply:
I think that was a mixup during git rebase!